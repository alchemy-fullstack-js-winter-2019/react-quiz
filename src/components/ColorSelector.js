import React from 'react';
import PropTypes from 'prop-types';



function ColorSelector({ handleChange }) {

    return (
        <>
            <label>FONT COLOR 
                <input type='color' name="FontColor" onChange={handleChange}/>
            </label>
            <label>BACKGROUND COLOR 
                <input type='color' name="BackgroundColor" onChange={handleChange}/>
            </label>
        </>
    );

}

ColorSelector.propTypes = {
    handleChange: PropTypes.func.isRequired
};


export default ColorSelector;
