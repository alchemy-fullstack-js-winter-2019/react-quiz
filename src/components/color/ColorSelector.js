import React, { Fragment } from 'react';

function ColorSelector() {
  return (
    <Fragment>
      <h2>I am colors</h2>
    </Fragment>
  );
}

export default ColorSelector;
