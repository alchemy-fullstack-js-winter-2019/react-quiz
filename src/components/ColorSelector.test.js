import React from 'react';
import renderer from 'react-test-renderer';
import ColorSelector from './ColorSelector';


describe('ColorSelector Test', () => {
    it('matches a snapshot', () => {
        const tree = renderer.create(
            <ColorSelector handleChange={() => {}}/>
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
