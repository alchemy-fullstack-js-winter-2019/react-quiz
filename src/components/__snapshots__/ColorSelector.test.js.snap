// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorSelector Test matches a snapshot 1`] = `
Array [
  <label>
    FONT COLOR
    <input
      name="FontColor"
      onChange={[Function]}
      type="color"
    />
  </label>,
  <label>
    BACKGROUND COLOR
    <input
      name="BackgroundColor"
      onChange={[Function]}
      type="color"
    />
  </label>,
]
`;
